mitxpro/templates/base.html

```diff
@@ -26,6 +26,7 @@
     {% block extrahead %}
     {% endblock %}
     {% endspaceless %}
+      <script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=8ef9ef96-3317-40a9-8ef6-de0737503caa"> </script>
```

Comment:
@asadiqbal08 do you know what this key does? Does it specify the account _and_ the brand? 

I don't usually like to hard code keys. Can we move this into a setting?